osu.Framework/Graphics/FrameworkColour.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK.Graphics;
+
+namespace osu.Framework.Graphics
+{
+    public static class FrameworkColour
+    {
+        public static Color4 Blue => new Color4(40, 65, 82, 255);
```

Comment:
```suggestion
        public static Color4 Blue { get; } = new Color4(40, 65, 82, 255);
```

maybe feels better? not sure it changes much. @smoogipoo opinions?

Reply:
What are you trying to achieve?

Reply:
Just asking your preferred code style.